ros_buildfarm/templates/misc/check_agents_job.xml.em

```diff
@@ -12,6 +12,12 @@
     'property_job-priority',
     priority=20,
 ))@
+@(SNIPPET(
+    'property_rebuild-settings',
+))@
+@(SNIPPET(
+    'property_rebuild-settings',
+))@
```

Comment:
Twice?

Reply:
Fixed in 8feb2ffea452ae1b434837d7ab469d9a8d1bdfed